code/src/Urls.js

```diff
@@ -0,0 +1,8 @@
+
+api_key = 'de0e6f6f34bb3f7060237da93b37b9e0';
+
+URL_POPULAR = `https://api.themoviedb.org/3/movie/popular?api_key=${api_key}&language=en-US&page=1`;
+
+PosterImage = `https://image.tmdb.org/t/p/w342/${movie.poster_path}`;
+
+BackdropImage = `https://image.tmdb.org/t/p/w1280/${movie.backdrop_path}`;
```

Comment:
Nice that you've gathered your url:s in a separate component! looks like a clean and organized approach! just wondering if you can import those url:s somehow in to your movie-components? if you declare and export them in this file maybe? or is the purpose of the url-component more of an overview of url:s needed in the project?

The api key is in api.js as well. do you need to have them in both components?

Reply:
Two issues in this file :

1. You forgot to use `const`/`let`/`var` keyword and because of that you accidentally created global variables (available in every moment in your code). This is huge antipattern, let's fixed that
2. You also forgot about `export` keyword - without it, we will not be able to import anything